plugins/kotlin/uast/uast-kotlin-fir/testData/legacyTypes/SimpleAnnotated.types.fir.txt

```diff
@@ -15,8 +15,5 @@ UFile (package = ) [public final class SimpleAnnotated {...]
                     UIdentifier (Identifier (println)) [UIdentifier (Identifier (println))]
                     USimpleNameReferenceExpression (identifier = println, resolvesTo = null) [println] : PsiType:Unit
```

Comment:
This `Unit` v.s. `void` in FE 1.0 UAST is only difference for type test. That is, I expect #1690 can make this identical too. Whichever goes first, I can rebase the remaining one.

Reply:
Okay, maybe bigger ones can go first. :)

Reply:
Yup, indeed identical on top of #1690